### PR TITLE
Feature email-to-buy

### PR DESCRIPTION
**Pull request by karsalfrink**
Let me know if this works for you and if so we can merge and I will go into Snipcart and turn off the service.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,12 +65,7 @@
 
       <div class="panel radius text-center">
         <p><em>Bycatch</em> is now available for <strong>$14</strong> plus shipping and tax.<br>We accept <strong>PayPal</strong> or <strong>Stripe</strong> and offer <strong>international shipping</strong>.<br>Supply is limited, so order your copy now!</p>
-        <a href="#" class="button radius snipcart-add-item"
-        data-item-id="4"
-        data-item-name="Bycatch card game"
-        data-item-price="14.00"
-        data-item-weight="163"
-        data-item-url="http://bycat.ch"><i class="fa fa-shopping-cart fa-lg"></i> Order</a>
+        <a href="mailto:bycatch@hubbub.eu?subject=Bycatch order" class="button radius"><i class="fa fa-shopping-cart fa-lg"></i>  Email to order</a>
```
Comment (alper) [inline index.html]: How would they know what to e-mail us? Or they send us an e-mail and we have a template response for it to figure out how many copies and how to pay us and stuff?

**Comment by karsalfrink** [comment]
They tell us how many they want. We send them an invoice and a payment request through PayPal or Stripe. Once they've payed we ship the copies. Cumbersome but I am willing to handle it.

**Comment by alper** [comment]
Could you prefill those questions into the e-mail body in some way so we get some information like that up front? But otherwise, that seems fine for the current order volume.

**Comment by karsalfrink** [comment]
Good point, let me see…

**Comment by karsalfrink** [comment]
How's that?

**Comment by alper** [comment]
:ship-it:

**Pull request merged**